datastore/grantserver/postgres.go

```diff
@@ -188,6 +188,8 @@ func NewPostgres(
 	// set max open connections to default to 80 (will get overwritten later by calculation
 	// depending of if we have environment variables set
 	db.SetMaxOpenConns(80)
+	// 50% of max open
+	db.SetMaxIdleConns(40)
```

Comment:
You could set maxOpenConns as a variable and set idle conns as function of that so you don't need to set in two places

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/855/commits/f4227ac45e5c93160e5422d948410a113972baba